src/Node.ts

```diff
@@ -2592,6 +2592,22 @@ export abstract class Node<Config extends NodeConfig = NodeConfig> {
   }
 }
 
+interface animOptions {
```

Comment:
I don't think we should define every property. We can just extend `ShapeConfig`, `ContainerConfig`, add some tween specific properties like "duration" and ability do define any custom property.

Reply:
Thanks for taking a look. My motivation for the type was to get some intellisense and type checking inside the to method to help understand what valid options are.

I'll take a look at your suggestion thanks, it'll be good practice for me.

Reply:
Hi Anton, 

this is just an idea based on your suggestion and incomplete, but is this more appropriate?

for pick type, i only picked types i thought were relevant to animation. 

This is just idea, i am only hoping for intellisense for the user.

//pick from nodeConfig those relating to animation
type animToPick = Pick<NodeConfig, "x" | "y" | "width" | "height" | "opacity" | "scaleX" | "scaleY" | "rotation" | "rotationDeg" | "offsetX" | "offsetY">
interface nodeAndFinish {
  // the values that failed travis tests
  node: Node;
  onfinish: Function;
}

interface animToSpecific {
  //and other possible methods / properties go here 
  duration?: number;
}

type animTo = animToPick & nodeAndFinish & animToSpecific;

Thank you

Reply:
Yes, something like that. But I don't think you need `Pick`, you can animate almost any property.